install

```diff
@@ -52,7 +52,7 @@ build_freorg_bkwdcomp=false
 # check if we have a modern version of getopt that can handle whitespace and long parameters
 getopt -T
 if [[ $? -eq 4 ]]; then
-    GETOPT_PARSE=$(getopt --name "${0}" --longoptions help,install,clients,debug,hip-clang,test,benchmark,package,relocatable,relwithdebinfo,codecoverage,address-sanitizer,rm-legacy-include-dir --options hicdtbpgrk -- "$@")
+    GETOPT_PARSE=$(getopt --name "${0}" --longoptions help,install,clients,debug,hip-clang,test,benchmark,package,relocatable,relwithdebinfo,static,codecoverage,address-sanitizer,rm-legacy-include-dir --options hicdtbpgrk -- "$@")
```

Comment:
you need to add 's' to '--options hicdtbpgrk' as well.

Reply:
Done in 
[Include new static option in install](https://github.com/ROCm/rocPRIM/pull/528/commits/f90715de8d07c38b487da85190db61664cdd976a)